samtranslator/plugins/application/serverless_app_plugin.py

```diff
@@ -308,6 +316,7 @@ def on_after_transform_template(self, template):
             while (time() - start_time) < self.TEMPLATE_WAIT_TIMEOUT_SECONDS:
                 temp = self._in_progress_templates
                 self._in_progress_templates = []
+                throttled = False
```

Comment:
Should we introduce some jitter? 

Should we update our Boto config to be [standard](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html#standard-retry-mode) (default is legacy)? Seems like that provides a better default retry behavior and I think standard is the newer recommendation.

Reply:
Yes, we should -- I'd suggest doing that in a future PR so we're introducing one change at a time.  I might have misread the botocore code but I thought even the legacy retry had some [jitter](https://github.com/boto/botocore/blob/23ee17f5446c78167ff442302471f9928c3b4b7c/botocore/data/_retry.json#L96) built into it.